docs/aetest/debugging.rst

```diff
@@ -486,3 +487,44 @@ Below are some examples of the pause on phrase feature in action:
     #   -> /path/to/my/example/testscript.py[20] test()
     #   (press Ctrl-D to resume execution)
     #   >>>
+
+The pause will also dump the connection information, including which devices are currently connected.
+
+.. code-block:: python
+
+    # --------------------------------------------------------------------------------
+    # Pause On Phrase: Connection Information
+    #
+    # +-------------------+--------------------------------+
+    # |  Device Property  |             Value              |
+    # +-------------------+--------------------------------+
+    # |        Name       |           device.name          |
+    # |       Alias       |           device.alias         |
+    # | Active Connection |   alias of active connection   |
+    # |       Status      |     Connected|Disconnected     |
+    # |   Spawn Command   |  command for active connection |
+    # |      --------     |            --------            |
+    # |  Connection Alias |   connection alias e.g. cli    |
+    # |       Class       |  Unicon, Gnmi, Netconf, etc    |
+    # |         IP        |        IPV4|IPV6 address       |
+    # |      Protocol     |        ssh, telnet, etc        |
+    # |      --------     |            --------            |
+    # |        URL        |    <protocol>://<ip>:<port>    |
+    # +-------------------+--------------------------------+
+
+
+If it is required to connect to the device directly while the test is paused, the device connection must be disconnected
+from before it can be accessed directly.
```

Comment:
```suggestion
.. note::
    
    If it is required to connect to the device directly while the test is paused, the device connection must be disconnected
    from before it can be accessed directly.
```
Maybe?